### PR TITLE
Update localization document with image tool

### PR DESCRIPTION
**Pull request by yiqunw700**
Fixes: #1659

### CODE REVIEW

File: docs/_docs/localization/index.md [hunk 1]
```diff
@@ -7,20 +7,26 @@ nav_order: 12
 # Localization
 
 We are developing tools for developers to localize the figma design into different languages more easily.
+
 The first introduced tool is a figma plugin that generates a strings xml file containing the strings
 used by the figma design.
 
+The second tool is a figma plugin that literates all the images, let user assign a proper name and
```
Comment (rylin8) [inline docs/_docs/localization/index.md:14]: Is "literates" supposed to be "iterates"?

File: docs/_docs/localization/index.md [hunk 2]
```diff
@@ -7,20 +7,26 @@ nav_order: 12
...
+download selected images as a `drawable.zip` file.
```
Comment (rylin8) [inline docs/_docs/localization/index.md:15]: Suggestion: "The second tool is a figma plugin that iterates through all the images, then lets a user assign a proper name and download selected images as a `drawable.zip` file."

File: docs/_docs/localization/index.md [hunk 3]
```diff
@@ -7,20 +7,26 @@ nav_order: 12
...
+
 ## 1. Run the localization plugin
 
-We have published our first version of the localization plugin. You can run it through:
-**Figma Main Menu -> Plugins -> From Google**
-**-> Automotive Design For Compose -> Localization Tool -> Generate String Resource**
+Both tools are now published public as part of the figma plugin [Automotive Design For Compose][1].
```
Comment (rylin8) [inline docs/_docs/localization/index.md:19]:
```suggestion
Both tools are now published publicly as part of the figma plugin [Automotive Design For Compose][1].
```

File: docs/_docs/localization/index.md [hunk 4]
```diff
@@ -40,25 +46,28 @@ Below are the steps to build the plugin. You can also find instructions at: <htt
...
+## 2. String resource tool
+
+### 2.1 Generate string resources
+
+#### 2.1.1 Strings generation options
 
 Currently the plugin provides three options to exclude text nodes from translations.
 
-<img src="./localization/LocalizationOptions.png">
+<img src="./LocalizationOptions.png">
```
Comment (rylin8) [inline docs/_docs/localization/index.md:70]: I remember an issue with the public site when these images were not in the proper folder, do you know if that's still the case?

File: docs/_docs/localization/index.md [hunk 5]
```diff
@@ -212,25 +222,85 @@ strings xml file and will not be fetched by DesignCompose.
...
+## 3. Image resource tool
+
+Images play an important role in the figma design. Figma supports PNG, JPG and GIF files up to 4k
+in width and height as background fills or strokes. In some cases, images need localization too,
+for example, the stop sign looks different in different countries.
+
+As a supplemental tool to the string resource tool, it works in a similar way that the plugin goes
+through all the images, allows user to assign a proper resource name to the image which in the
+Android resource framework is the image file name and it writes the resource name to the figma
+plugin data which the DesignCompose uses later to load the image resource.
```
Comment (rylin8) [inline docs/_docs/localization/index.md:240]: Suggestion: "As a supplemental tool to the string resource tool, the image resource tool works in a similar way. It goes through all the images and allows a user to assign a proper resource name to the image to match the image file name in the Android resource framework. It then writes the resource name to the Figma plugin data, which DesignCompose uses later to load the image resource."

File: docs/_docs/localization/index.md [hunk 6]
```diff
@@ -212,25 +222,85 @@ strings xml file and will not be fetched by DesignCompose.
...
+
+### 3.1 Download images
+
+Run `Export Images` under the `Localization Tool`. The plugin will display a list of images used by
+the figma design. Each image will have `Image hash`, `Image res name`, the image and a list of
```
Comment (rylin8) [inline docs/_docs/localization/index.md:245]:
```suggestion
the Figma design. Each image will have `Image hash`, `Image res name`, the image and a list of
```

File: docs/_docs/localization/index.md [hunk 7]
```diff
@@ -212,25 +222,85 @@ strings xml file and will not be fetched by DesignCompose.
...
+`node id`s which are clickable to show the node with the image.
+
+<img src="./ImageTool.png">
+
+Each image has a checkbox which is unchecked by default. When checked, the image resource name will
+be written to the shared plugin data with a specific namespace fetched by DesignCompose and the
+image will be exported when download.
```
Comment (rylin8) [inline docs/_docs/localization/index.md:252]:
```suggestion
image will be exported when downloaded.
```

File: docs/_docs/localization/index.md [hunk 8]
```diff
@@ -212,25 +222,85 @@ strings xml file and will not be fetched by DesignCompose.
...
+The image res name is an edit box which allows you to rename the auto created image resource name
+to a more meaningful name. Note: you need to check the destination drawable folder you're planning
+to place the images manually and avoid using the same resource name if the images are different.
+
+After reviewing, renaming and picking the images for localization, you can click the `Download`
+button to download the selected images as the `drawable.zip` file.
+
+<img src="./SaveImage.png">
+
+Note: Although this tool can export GIF images, DesignCompose doesn't support GIF now. Figma REST api
```
Comment (rylin8) [inline docs/_docs/localization/index.md:263]:
```suggestion
Note: Although this tool can export GIF images, DesignCompose currently doesn't support GIF. Figma's REST API
```

File: docs/_docs/localization/index.md [hunk 9]
```diff
@@ -212,25 +222,85 @@ strings xml file and will not be fetched by DesignCompose.
...
+returns both `imageRef` and `gifRef` but DesignCompose only renders the `imageRef` which is a static
+image(see `Paint` properties <https://www.figma.com/developers/api#files-types>). The plugin api
```
Comment (rylin8) [inline docs/_docs/localization/index.md:265]:
```suggestion
image(see `Paint` properties <https://www.figma.com/developers/api#files-types>). The plugin API
```

File: docs/_docs/localization/index.md [hunk 10]
```diff
@@ -212,25 +222,85 @@ strings xml file and will not be fetched by DesignCompose.
...
+can only get the image hash(i.e. the `gifRef`), so localization doesn't work for GIFs at this point.
+
+### 3.2 Integrate with the app and test
+
+To integrate with the app, unzip the `drawable.zip` file and place all the images under the app's
+`res/drawable` folder. For each of image, place localized images under `res/drawable-[LOCALE]` folder.
```
Comment (rylin8) [inline docs/_docs/localization/index.md:271]:
```suggestion
`res/drawable` folder. For each of the images, place localized images under `res/drawable-[LOCALE]` folder.
```

File: docs/_docs/localization/index.md [hunk 11]
```diff
@@ -212,25 +222,85 @@ strings xml file and will not be fetched by DesignCompose.
...
+When the device's locale changes to Simplified Chinese, the app should use the localized image for
+the stop sign from `drawable-zh`.
+<img src="./ImageLocalizedResult.png">
+
+Also, similar to 2.2.3, we can re-enable the live update for images by unchecking the
+`Use local resources if exists` option in design switcher.
+
+### 3.3 What is not supported
+
+Android recommend to create different versions of images for different screen densities. This tool
```
Comment (rylin8) [inline docs/_docs/localization/index.md:287]:
```suggestion
Android recommends creating different versions of images for different screen densities. This tool
```

File: docs/_docs/localization/index.md [hunk 12]
```diff
@@ -212,25 +222,85 @@ strings xml file and will not be fetched by DesignCompose.
...
+only download the images in their original sizes and we suggest acquiring the other scales of
```
Comment (rylin8) [inline docs/_docs/localization/index.md:288]:
```suggestion
only downloads the images in their original sizes and we suggest acquiring the other scales of
```

File: docs/_docs/localization/index.md (hunk 1 above)
Comment (yiqunw700) [reply docs/_docs/localization/index.md]: Yes. 😅

File: docs/_docs/localization/index.md (hunk 4 above)
Comment (yiqunw700) [reply docs/_docs/localization/index.md:71]: I moved the md file under the folder. That is why I don't need this any more.

**Review by rylin8 — APPROVED** [review]

**Pull request merged**